style.css

```diff
@@ -0,0 +1,97 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #e2e8f0;
+}
+
+.container #title {
+  font-size: 2.5em;
+  font-weight: 600;
+  margin-bottom: 1em;
+  text-align: center;
+}
+
+.container #description {
+  font-size: 1.2em;
+  font-weight: 400;
+  margin-bottom: 2em;
+  text-align: center;
+}
+
+.container form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-color: #03012c;
+  color: #fff;
+  padding: 2em;
+  border-radius: 5px;
+}
+
+.container form label {
+  font-size: 1.2em;
+  font-weight: 600;
+}
+
+.container form input {
+  padding: 0.5em 2rem;
+  margin-bottom: 1em;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+}
```

Comment:
[ OPTIONAL ]

Please, I suggest using CSS Selectors in your codes 👨‍💻 , making your code clean and dry.